openapi2beans/JavaChecker/src/main/java/dev/galasa/openapi2beans/example/generated/AnEnumProperty.java

```diff
@@ -0,0 +1,7 @@
+package dev.galasa.openapi2beans.example.generated;
```

Comment:
Please check that copyright is added to all the files. I'll only make one comment here, but it needs re-checking and fixing for all files please.

Reply:
Should generated files have copyright added too ? I think so yes.

Reply:
added copywriting in generated code